syntax = "proto3";

// MPF will request changes at startup and then stream the changes
message SwitchChangesRequest {
}

// Platform streams switch changes to MPF
message SwitchChanges {
  string switch_number = 1;
  bool switch_state = 2;
  uint64 change_time = 3;
}
